ci-automation/testing_env_main.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+homedir=`dirname "${0}"`
+command=$1
+
+on_destroy() {
+  rm -rf /var/tmp/CentOS-Dockerfiles
+  docker rm -f centos-test-env
+  sudo rm -rf $homedir/.vagrant/machines/default/libvirt
+}
+
+on_exit() {
+    echo "$?"
+    on_destroy
+    sudo rm -rf $homedir/.vagrant/machines/default/libvirt
+    exit
+}
+
+build_images() {
+  pushd /var/tmp
+  # cloning private account repo till PR is approved and merged
+  #git clone https://github.com/CentOS/CentOS-Dockerfiles.git
+  git clone https://github.com/lioramilbaum/CentOS-Dockerfiles.git
+  cd CentOS-Dockerfiles/systemd/centos7
+  docker build -t centos/systemd .
```

Comment:
why not just pull this image from the CentOS registry instead of building it on your own?

generally git cloning other repos in your CI script is a bad idea - you introduce 3rd party breakage as noise to your CI that way...

Reply:
I didn't find the CentOS registry to pull the image from.

Reply:
centos/systemd is found. Removed the image building steps.